caffe2/CMakeLists.txt

```diff
@@ -1013,6 +1013,9 @@ elseif(USE_CUDA)
 endif()
 
 if(USE_XPU)
+  if(USE_XCCL)
+    append_filelist("libtorch_xpu_distributed_extra_sources" Caffe2_XPU_SRCS)
```

Comment:
The purpose of the `append_filelist` function is to append the files listed in the first variable to the second variable. `libtorch_xpu_distributed_extra_sources` is defined in `build_variables.bzl`. The reason `ProcessGroupXCCL.cpp` is added to this list instead of being directly appended to the `Caffe2_XPU_SRCS` variable is that we plan to add more utility files to the list later.